templates/agenda_case_item/documents.html.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'layout-with-agenda-case-item-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda_item' %}
```

Comment:
I'm usually in favour of splitting things up, but maybe this should be covered by the agenda translation domain?

Reply:
Merged `agenda_item` translation domain into `'agenda`. Also updated some translations.